app/controllers/api/user_programs_controller.rb

```diff
@@ -1,9 +1,10 @@
 class Api::UserProgramsController < ApplicationController
   before_action :authenticate_user!
 
-
   def index(page: nil)
     @programs = current_user.programs.unchecked
+                  .work_published
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.